cmd/bnscli/cmd_cash.go

```diff
@@ -78,13 +78,13 @@ already has a fee set, overwrite it with a new value.
 	}
 
 	if coin.IsEmpty(amountFl) {
-		msg, err := tx.GetMsg()
-		if err != nil {
-			return fmt.Errorf("cannot extract message from transaction: %s", err)
+		msg, msgErr := tx.GetMsg()
```

Comment:
Hmmm... so we need to define a separate err variable for each line?
I guess it enforces that we check each one of them, but does seem a bit verbose.
err, err2, err3, err4...

Reply:
We don't need to define a separate error as long as we don't shadow it. So that basically means we either need to define all the variables like `msg` upfront or indeed use a new name for each error which is used alongside a new variable definition.